public/locales/en-US/send.ftl

```diff
@@ -16,9 +15,9 @@ importingFile = Importing…
 verifyingFile = Verifying…
 encryptingFile = Encrypting…
 decryptingFile = Decrypting…
-notifyUploadDone = Your upload has finished.
+notifyUploadEncryptDone = Your file is encrypted and ready to send.
```

Comment:
The 'send' verb here is awkward to me. 'ready to send' sounds like nothing has happened yet. I'd write it as 'Your file is uploaded and ready to share'.